test/e2e/common/setup-kubestellar.sh

```diff
@@ -197,4 +197,4 @@ if ! expect-cmd-output 'kubectl --context its1 get managedclusters -l location-g
 then
     echo "Failed to see two clusters."
     exit 1
-fi
+fi
```

Comment:
Please do not delete the linebreak at the end of this file. Same for the other file.

<img width="102" height="37" alt="Image" src="https://github.com/user-attachments/assets/718ba0eb-75c2-4ada-b836-301226bf2750" />